app/assets/javascripts/components/_calendar.js.jsx

```diff
@@ -23,11 +23,18 @@ var Calendar = React.createClass({
 
   render: function () {
     var name = this.props.week[this.state.day];
+    var list;
+    if (this.state.day === 'weekend') {
+      list = '';
+    } else {
+      list = <ChoreList className="chore-list" day={ this.state.day }/>
+    }
+
     return (
       <div className={["calendar", [this.state.day, "color"].join("-")].join(" ")}>
         <div className="calendar-inner">
           <Week className="week" days={ this.props.week } onSetDay={this.setDay}/>
-          <ChoreList className="chore-list" day={ this.state.day }/>
+          { list }
```

Comment:
@moneypenny here. You may remember me from such PRs as "200,000 lines of glory" and "let me just :nail_care: a little here." Another way you could do this:

```javascript
{
  this.state.day === 'weekend' ? '' : (
    <ChoreList day={ this.state.day }/>
  )
}
```

I wouldn't pass the class 'chore-list' to the component, I'd just put that hardcoded in the component since its name is ChoreList.